src/core/pom.xml

```diff
@@ -15,6 +15,21 @@
   <name>GeoGig Core</name>
   <!-- Build Instructions and Profiles Handled as a normal maven java project: mvn clean install Online tests available using:
     mvn -Ponline Corertura is configured for a test coverage report: mvn cobertura:cobertura open target/site/cobertura/index.html -->
+
+  <licenses>
+    <license>
+      <name>Eclipse Distribution License</name>
+      <url>https://www.eclipse.org/org/documents/edl-v10.html</url>
+      <distribution>repo</distribution>
+      <comments>The Eclipse Distribution License (a BSD-3 style license)</comments>
+    </license>
+    <license>
+      <name>Apache License, Version 2.0</name>
+      <url>https://www.apache.org/licenses/LICENSE-2.0.txt</url>
+      <distribution>repo</distribution>
+      <comments>This license only applies to varint and Google diff merge patch</comments>
+    </license>
+  </licenses>
```

Comment:
I considered writing this one up as a third-party dependency (in a NOTICE file). The code has been cut and pasted in, I am not sure if each license entry in a pom.xml is viewed as a dual-license (in which case this pull-request is wrong) or if we are listing all the licenses used (in which case this is correct).

The comment is there to clarify.